src/backend/controllers/tankController.js

```diff
@@ -1,68 +1,64 @@
 // @flow strict
 
-// Flow type import
-import type { Request, Response } from 'express';
-
 const User = require('../../models/userModel');
 const Tank = require('../../models/tankModel');
 
 const { validationResult } = require('express-validator');
 
-exports.getFavorite = async (req: Request, res: Response) => {
+// Flow type import
+import type {
+	$Request,
+	$Response,
+	NextFunction,
+	Middleware,
+} from 'express';
```

Comment:
So is there a reason why it was moved here rather just just appended to the previous implementation?

Also is there a reason why you need `NextFunction` and `Middleware` if these imports are not used at all?